pyproject.toml

```diff
@@ -107,7 +107,7 @@ test-command = "pytest {package}/tests"
 #    PyPy
 #    musllinux when python < 3.9
 #    musllinux in aarch64
-skip = ["pp*", "cp38-musllinux*", "*-musllinux_aarch64"]
+skip = ["pp*", "cp39-musllinux*", "*-musllinux_aarch64"]
```

Comment:
please read the comment 😬

Reply:
We can now remove `"cp38-musllinux*"`. The `skip` will be joined with `python >=`. If we have `python >= 3.9`, any `3.8` build will be skipped anyway.